docs/conf.py

```diff
@@ -18,7 +18,7 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-sys.path.insert(0, os.path.abspath("../src"))
+# sys.path.insert(0, os.path.abspath("../src"))
```

Comment:
?

Reply:
If ``../src`` is prepended to sys.path, the runtime would try to import the low-level _sodium bindings from a nonexistent file in src/nacl ignoring the one installed in site-packages by pip/setup.py; for a comparison, see:
```
$ ./venv-35/bin/python -c 'import sys; import os; sys.path.insert(0, os.path.abspath("./src"));from nacl._sodium import ffi, lib; print("OK")'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named 'nacl._sodium'
```
vs
```
$ ./venv-35/bin/python -c 'from nacl._sodium import ffi, lib; print("OK")'
OK
```

Reply:
Sorry, I meant that if this is not necessary/desirable we should delete it rather than commenting it out.

In cryptography's config this is just `os.path.abspath(".")` and is set that way so that we can import a custom extension that is in the same dir.

Reply:
I'd prefer restoring the comment to the same content that was generated by sphinx-quickstart.
Let me know if this can make sense.